bin/nginx_release_downloads.sh

```diff
@@ -14,22 +14,28 @@
 
 downloads=https://nginx.org/download
 
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  SED='gsed'
+else
+  SED='sed'
+fi
```

Comment:
Per my comment above, consider this instead:
```shell
if [ "$(uname)" = 'Darwin' ]; then
  SED='gsed'
else
  SED='sed'
fi
```

Reply:
Ack, addressed in https://github.com/DataDog/nginx-datadog/pull/16/commits/57c3eedde296169327ad1d20b9b2d395b073a086